pkg/config/context.go

```diff
@@ -147,6 +147,13 @@ func (c *Context) GetState() *ContextState {
 	return nil
 }
 
+func (c *Context) GetConfig() *Config {
+	if c != nil {
+		return c.Config
+	}
+	return nil
+}
```

Comment:
Not a fan of adding this. If anything, we should try to remove config as a field of context (to remove the circular dependency) since config is the parent of context

Reply:
Good point. This should be unnecessary as well after applying your first comment. I'll test out all the cases again.

Reply:
Yup, still works!